.vscode/ruby-lsp/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'ruby-lsp'
+eval_gemfile(File.expand_path('../../Library/Homebrew/Gemfile', __dir__))
```

Comment:
```suggestion
source "https://rubygems.org"

gem "ruby-lsp"
eval_gemfile(File.expand_path("../../Library/Homebrew/Gemfile", __dir__))
```

Reply:
Also think this could be added to our main `Gemfile` in a specific `vscode` group?